client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -170,6 +180,62 @@ export default () => {
         {!participant && !error && <Alert severity='info'>Loading participant details</Alert>}
         {participant && (
           <Card>
+            {selectedCohort !== null && (
+              <Dialog
+                showDivider={true}
+                title='Assign Cohort'
+                open={selectedCohort !== null}
+                onClose={onClose}
+              >
+                <DialogContent>
+                  <Grid container spacing={4}>
+                    <Grid item xs={12}>
+                      <Grid container spacing={1}>
+                        <Grid item xs={4}>
+                          <Typography variant='body1'>Participant Name</Typography>
+                        </Grid>
+                        <Grid item xs={4}>
+                          <Typography variant='body2'>{participant.fullName}</Typography>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                    <Grid item xs={12}>
+                      <Grid container spacing={1}>
+                        <Grid item xs={4}>
+                          <Typography variant='body1'>Assign Cohort</Typography>
+                        </Grid>
+                        <Grid item xs={4}>
+                          <Typography variant='body2'>{cohortName(selectedCohort)}</Typography>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                  </Grid>
+                  <br />
+                  <Box>
+                    <Typography variant='body2'>
+                      Are you you going to assign this participant to{' '}
+                      <b>{cohortName(selectedCohort)}</b>. Please revise information before taking
+                      action
```

Comment:
Suggested Copy Change:
Are you sure that you would like to assign this participant to [cohortName]? Please review the above information before proceeding.

Reply:
OK, I copied it from design ...